CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+v0.11.0 - _TBD_
+------------------------
+    * Added `zeroEx.token.setUnlimitedProxyAllowanceAsync` (#137)
+    * Added `zeroEx.token.setUnlimitedAllowanceAsync` (#137)
+    * Added `zeroEx.token.UNLIMITED_ALLOWANCE_IN_BASE_UNITS` (#137)
```

Comment:
Let's rename this constant to `UNLIMITED_ALLOWANCE_AMOUNT`. It's become a library standard that amounts are always in base units.